Mathlib/Algebra/Order/Sub/Basic.lean

```diff
@@ -116,9 +118,11 @@ theorem tsub_eq_tsub_min (a b : α) : a - b = a - min a b := by
 
 namespace AddLECancellable
 
+omit [CanonicallyOrderedAdd α] in
 protected theorem lt_tsub_iff_right (hc : AddLECancellable c) : a < b - c ↔ a + c < b :=
   ⟨lt_imp_lt_of_le_imp_le tsub_le_iff_right.mpr, hc.lt_tsub_of_add_lt_right⟩
 
+omit [CanonicallyOrderedAdd α] in
 protected theorem lt_tsub_iff_left (hc : AddLECancellable c) : a < b - c ↔ c + a < b :=
   ⟨lt_imp_lt_of_le_imp_le tsub_le_iff_left.mpr, hc.lt_tsub_of_add_lt_left⟩
```

Comment:
No problem if it is a pain, but if it is possible to rearrange the order of theorems here so the omit isn't necessary that would be nice.